### ISSUE TITLE
Pretraining details

### ISSUE THREAD

**Issue by lucadiliello** [idb-ita/GilBERTo#6]
Would like to know more on the pretraining hyper-parameters. Which batch size, learning rate, scheduler, optimizer, ... did you use?
